globals.go

```diff
@@ -327,6 +327,13 @@ var (
 		TriggerRequester,
 	}
 
+	// all types that can request versions to be on the waterfall
+	VersionRequesterTypes = []string{
```

Comment:
Did you look at the places using the system requester types and decide to split them up? I took a glance at them and it seems appropriate to add them there. It'll make tag versions show up on things like task history, notifications,  grouped with commits in scheduler, etc